cmd/aks-periscope/aks-periscope.go

```diff
@@ -14,6 +15,7 @@ import (
 
 func main() {
 	zipAndExportMode := true
+	//exporter := &exporter.LocalMachineExporter{}
```

Comment:
💡Nit: Is this intentional? might be removed.

Reply:
This is a new exporter I will eventually be submitting a PR for. I am planning to submit a PR for each new class I add in order to decouple my changes to your codebase.